### PR TITLE
fix(TDI-43010): [ADLSgen2] Short term solution for columns selector i…

### PR DESCRIPTION
**Pull request by slushatel**
[ADLSgen2] Short term solution for columns selector in Studio. Hide 'CSV schema'field

**What is the current behavior?** (You can also link to an open issue here)
'CSV schema' field is on the form

**What is the new behavior?**
'CSV schema' field is hidden

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java [hunk 1]
```diff
@@ -201,6 +202,15 @@ private void buildActivationCondition(final PropertyNode node, final PropertyNod
      */
     @Override
     public void visit(final PropertyNode node) {
+        // skip 'configuration.dataSet.csvConfiguration.csvSchema' field in 'azure-dls-gen2' component
+        if (element != null && element instanceof Node && ((Node)element).getComponent() != null &&
+                ((Node)element).getComponent().getName() != null &&
+                ((Node)element).getComponent().getName().equals("AzureAdlsGen2Input") &&
+                node != null && node.getProperty() != null && node.getProperty().getPath() != null &&
```
Comment (gonchar-ivan) [inline main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java]: please remove 
```
node != null && node.getProperty() != null && node.getProperty().getPath() != null
```
I'm pretty sure that this variables are not null.

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java [hunk 2]
```diff
@@ -201,6 +202,15 @@ private void buildActivationCondition(final PropertyNode node, final PropertyNod
      */
     @Override
     public void visit(final PropertyNode node) {
+        // skip 'configuration.dataSet.csvConfiguration.csvSchema' field in 'azure-dls-gen2' component
+        if (element != null && element instanceof Node && ((Node)element).getComponent() != null &&
+                ((Node)element).getComponent().getName() != null &&
```
Comment (gonchar-ivan) [inline main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java]: Please, remove
```
 ((Node)element).getComponent().getName() != null
```
 ((Node)element).getComponent().getName() != null is always not null

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java (hunk 1 above)
Comment (slushatel) [reply main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java]: removed

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java (hunk 2 above)
Comment (slushatel) [reply main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java]: removed

**Pull request merged**